src/file-1.xoo

```diff
@@ -0,0 +1,25 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// dummy line 7
+// dummy line 8
+// dummy line 9
+// dummy line 10
+// dummy line 11
+// dummy line 12
+// dummy line 13
+// dummy line 14
+// dummy line 15
+// dummy line 16
+// violate HasTag rule: xoo on line 17
+// dummy line 18
+// dummy line 19
+// dummy line 20
+// dummy line 21
+// dummy line 22
+// dummy line 23
+// dummy line 24
+// violate HasTag rule: xoo on line 25
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:3000/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:3000/project/issues?id=sample&branch=feature1&issues=AWBvYcRO5knRPmAAmJqw&open=AWBvYcRO5knRPmAAmJqw)

[ISSUE_KEY]: AWBvYcRO5knRPmAAmJqw